apps/common/send_email.py

```diff
@@ -0,0 +1,40 @@
+from fastapi_mail import FastMail, MessageSchema, ConnectionConfig
+from settings import Settings
+from typing import List
+
+
+conf = ConnectionConfig(
+    MAIL_USERNAME=Settings.MAIL_USERNAME,
+    MAIL_PASSWORD=Settings.MAIL_PASSWORD,
+    MAIL_FROM=Settings.MAIL_FROM,
+    MAIL_PORT=Settings.MAIL_PORT,
+    MAIL_SERVER=Settings.MAIL_SERVER,
+    MAIL_FROM_NAME=Settings.MAIL_FROM_NAME,
+    MAIL_TLS=Settings.MAIL_TLS,
+    MAIL_SSL=Settings.MAIL_SSL,
+    USE_CREDENTIALS=Settings.USE_CREDENTIALS,
+    TEMPLATE_FOLDER=Settings.TEMPLATE_FOLDER,
+)
+
+
+async def send_email_async(
+        subject: str,
+        email_to: List[str],
+        body: dict,
+        template_name: str = 'email.html',
+) -> None:
```

Comment:
should change `None` to `Optional[dict]`, because `Optional[dict] = dict | None`
In this method documentation we can see what this method returns:

```
This method will return normally if the mail is accepted for at least one recipient. It returns a dictionary, with one entry for each recipient that was refused.
```